features/rules/ALB/ALB012_Alignment-vertical-segment-radius-of-curvature.feature

```diff
@@ -1,6 +1,6 @@
 @implementer-agreement
 @ALB
-@version1
+@version2
 @E00020
 Feature: ALB012 - Alignment vertical segment radius of curvature
   The rule verifies the 'RadiusOfCurvature' design parameter for vertical alignment segments.
```

Comment:
```suggestion
  The rule verifies the 'RadiusOfCurvature' design parameter for vertical alignment segments. Where specified, an absolute tolerance value in the length unit of the model is used for the comparison.
```

If I understand correctly, the tolerance is file units, so this means that an alignment in millimeters has to be more precise then an alignment in meters. I don't think this is wrong per se. The reverse is also odd to have a tolerance in meters that would be converted to feet for example. But it wouldn't hurt to be explicit about it.